src/main/java/com/rtb/projectmanagementtool/project/CreateProjectServlet.java

```diff
@@ -0,0 +1,55 @@
+package com.rtb.projectmanagementtool.project;
+
+import com.google.appengine.api.datastore.*;
+import com.rtb.projectmanagementtool.auth.*;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/create-project")
+public class CreateProjectServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    // Authenticate
+    AuthOps auth = new AuthOps(datastore);
+    auth.loginUser(request, response);
+    Long userLoggedInId = auth.whichUserIsLoggedIn(request, response);
+    if (userLoggedInId == Long.parseLong(AuthOps.NO_LOGGED_IN_USER)) {
+      response.sendRedirect("/login");
+      return;
+    }
+
+    // userId attribute set here so a new AuthOps object is not
+    // created in doPost() method
+    request.setAttribute("userId", userLoggedInId);
+
+    // Forward to create project page
+    request.getRequestDispatcher("create-project.jsp").forward(request, response);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response)
+      throws ServletException, IOException {
+
```

Comment:
Also check credentials here.  We wouldn't want them creating new projects if they're not logged in.